doc/guide/authentication.xml

```diff
@@ -99,4 +99,49 @@
 
     </section>
   </section>
+
+  <section id="2fa-auth">
+    <title>Enable two-factor authentication</title>
+
+    <para>As with any other <code>pam</pam> based service that supports it,
+      you can harden your Cockpit authentication by adding two-factor authentication.
+      This can be achieved by installing the <code>google-authenticator</code> package
```

Comment:
The package is called `libpam-google-authenticator` on Debian, and probably other things in other places.  Maybe specify it by its proper name and then give the example of the package name (enclosed in `<code>` tags) for a couple of popular distros?

Reply:
Indeed:

- Debian based OS: libpam-google-authenticator
- Fedora: google-authenticator (CentOS use it probably as well, will check)
- openSUSE: google-authenticator-libpam